README.md

```diff
@@ -148,44 +148,44 @@ seeds = "f4678480e4b7f1daee8ff47e6265954f8b57291d@54.234.180.96:26656,f6764b2a92
 ...
 ```
 
-Then copy the testnet genesis JSON file to the node configuration directory. The latest genesis file can be found [here](https://github.com/certikfoundation/chain/blob/master/genesis.json). Release-specific genesis file can be found in the [release download](https://github.com/certikfoundation/chain/releases/).
+Then copy the testnet genesis JSON file to the node configuration directory. The latest genesis file can be found [here](https://github.com/shentufoundation/chain/blob/master/genesis.json). Release-specific genesis file can be found in the [release download](https://github.com/shentufoundation/chain/releases/).
```

Comment:
the links here should be
https://github.com/shentufoundation/shentu/blob/master/genesis.json
https://github.com/shentufoundation/shentu/releases/